public/app/app.ts

```diff
@@ -236,18 +226,16 @@ export class GrafanaApp {
         await preloadPlugins(awaitedAppPlugins, pluginExtensionsRegistries, 'frontend_awaited_plugins_preload');
       }
 
-      setPluginExtensionGetter(
-        createPluginExtensionsGetter(
-          pluginExtensionsRegistries.extensionsRegistry,
-          pluginExtensionsRegistries.addedComponentsRegistry
-        )
-      );
-      setPluginExtensionsHook(
-        createUsePluginExtensions(
-          pluginExtensionsRegistries.extensionsRegistry,
-          pluginExtensionsRegistries.addedComponentsRegistry
-        )
-      );
+      try {
+        setPluginLinksHook(createUsePluginLinks(pluginExtensionsRegistries.addedLinksRegistry));
+      } catch (e) {
+        if (e instanceof Error) {
+          logError(e);
+        }
+        console.error(e);
+      }
```

Comment:
Hmm, why do we have separate error-handling for the `setPluginLinksHook()` and not fir the other registries below?